spec/actions/get_latest_github_release_witin_same_major_action_spec.rb

```diff
@@ -0,0 +1,48 @@
+describe Fastlane::Actions::GetLatestGithubReleaseWithinSameMajorAction do
+  describe '#run' do
+    let(:repo_name) { 'purchases-ios' }
+    let(:get_releases_purchases_ios_response) do
+      { json: JSON.parse(File.read("#{File.dirname(__FILE__)}/../test_files/get_releases_purchases_ios.json")) }
```

Comment:
I added a minified response from the github API for the tests so they are more thorough.